src/TesApi.Web/Management/Batch/MappingProfilePoolToWsmRequest.cs

```diff
@@ -46,7 +48,12 @@ public MappingProfilePoolToWsmRequest()
             CreateMap<BatchVmContainerRegistry, ApiContainerRegistry>()
                 .ForMember(dst => dst.IdentityReference, opt => opt.MapFrom(src => src.IdentityResourceId));
             CreateMap<BatchNetworkConfiguration, ApiNetworkConfiguration>()
-                .ForPath(dst => dst.EndpointConfiguration.InboundNatPools, opt => opt.MapFrom(src => src.EndpointInboundNatPools));
+                .ForMember(dst => dst.EndpointConfiguration, opt => opt.MapFrom(src => src.EndpointInboundNatPools));
+            CreateMap<IList<BatchInboundNatPool>, ApiEndpointConfiguration>()
+                .ForMember(dst => dst.InboundNatPools, opt => opt.MapFrom(src => src))
+                .ReverseMap();
+            CreateMap<ApiInboundNatPool[], IList<BatchInboundNatPool>>()
+                .ForMember("Item", opt => opt.Ignore());
```

Comment:
This is a fix for a problem with the shape of the WSM batch pool creation request.

Reply:
That same fix is in #764